Gini-iOS-SDK/GINIDocumentTaskManager.m

```diff
@@ -72,21 +72,29 @@ - (BFTask *)getDocumentWithId:(NSString *)documentId{
 }
 
 - (BFTask *)createDocumentWithFilename:(NSString *)fileName fromImage:(UIImage *)image {
-    NSParameterAssert([fileName isKindOfClass:[NSString class]]);
-    NSParameterAssert([image isKindOfClass:[UIImage class]]);
+    return [self createDocumentWithFilename:fileName fromImageData:UIImageJPEGRepresentation(image, 0.2)];
+}
+
+- (BFTask *)createDocumentWithFilename:(NSString *)fileName fromImage:(UIImage *)image docType:(NSString *)docType {
+    return [self createDocumentWithFilename:fileName fromImageData:UIImageJPEGRepresentation(image, 0.2) docType:docType];
+}
 
-    BFTask *createTask = [[_apiManager uploadDocumentWithData:UIImageJPEGRepresentation(image, 0.2) contentType:@"image/jpeg" fileName:fileName docType:nil] continueWithSuccessBlock:^id(BFTask *task) {
+- (BFTask *)createDocumentWithFilename:(NSString *)fileName fromImageData:(NSData *)imageData {
```

Comment:
`fromData:` would be better, PDFs or UTF-8 text can also be uploaded, even though content type is set as `image/jpeg`. The Android version does the same. Here are [api supported file formats](http://developer.gini.net/gini-api/html/documents.html#supported-file-formats).